lib/src/widgets/controls.dart

```diff
@@ -247,8 +297,8 @@ class ControlState extends State<Control> with SingleTickerProviderStateMixin {
                       ),
                     ),
                     Positioned(
-                      right: 15,
-                      bottom: 12.5,
+                      right: 16,
```

Comment:
why we don't keep old position?

Reply:
Adding an additional button certainly needs more space, infact this UI code wasn't written by me but a contributor. Ideally, one would write their own custom designed video controls. These "just work" and are very "generic" & here to show the plugin capabilities etc. in the example itself.